pxl_scripts/px/pod/pod.pxl

```diff
@@ -40,6 +39,21 @@ def containers(start_time: str, pod: px.Pod):
     return df
 
 
+def node(start_time: str, pod: px.Pod):
+    ''' A list containing the node the `pod` is running on.
+
+    Args:
+    @start_time: The timestamp of data to start at.
+    @pod: The name of the pod to filter on.
+
+    '''
+    df = px.DataFrame(table='process_stats', start_time=start_time)
+    df = df[df.ctx['pod'] == pod]
+    df.name = df.ctx['node_name']
+    df = df.groupby(['name']).agg()
```

Comment:
nit: you can omit the bracket and just say 'name' when the list only has one element